internal/view/svc.go

```diff
@@ -60,6 +60,10 @@ func (s *Service) showPods(a *App, _ ui.Tabular, gvr, path string) {
 		return
 	}
 
```

Comment:
How about short circuiting this right off the bat? ie these 3 lines should replace this entire PR
```go
	if svc.Spec.Type == v1.ServiceTypeExternalName {
		return
	}
```